docs/using-github-extensions.md

```diff
@@ -18,7 +18,7 @@ Extensions, apps, or bots that can **arbitrarily modify code are NOT allowed**.
 
 ## Writing your GitHub Action workflows
 
-Many GitHub Action workflows do not require a dedicated GitHub account. Good examples are auto-assign workflows for issues and PRs such as [the one used in semantic convention repo](https://github.com/open-telemetry/semantic-conventions/blob/main/.github/workflows/auto-assign-issue.yml).
+Many GitHub Action workflows do not require a dedicated GitHub account. Good examples are auto-assign workflows for PRs such as [the one used in specification repo](https://github.com/open-telemetry/opentelemetry-specification/blob/main/.github/auto_assign_pr.yml).
```

Comment:
unrelated to this PR,  fixes CI failure - the workflow was recently removed in https://github.com/open-telemetry/semantic-conventions/pull/1497